### DISCUSSION TITLE
Calibration data file - what are them?

### DISCUSSION THREAD

**Discussion by fgdfgfthgr-fox** [turboderp/exllamav2#13] [Q&A] [answered]
It seems like we need a calibration data file to quantize models, but what are them? Where can we get them?
Thanks?

**Comment by turboderp** [comment, marked as answer]
Right, I'm going to do a little writeup on the quantizer clarifying all that. But in the meantime, it uses Parquet files for calibration data. You can find them all over HF, auto-converted from the various datasets there. For instance, [here is wikitext-test](https://huggingface.co/datasets/wikitext/tree/refs%2Fconvert%2Fparquet/wikitext-103-raw-v1/test), the same dataset that TheBloke seems to use for GPTQ conversions now.
